chalice/app.py

```diff
@@ -58,6 +58,25 @@ def dispatch(*args, **kwargs):
     #                             headers={"Content-Type": "text/html"},
     #                             body=swagger_ui_html)
 
+    @app.route("/version")
+    def version():
```

Comment:
Isn't version also encoded in the url?

Reply:
Following the pattern of other component version endpoints, this serves the version of code that is deployed, generated by `git describe --tags --always` which includes the hash of the commit and the latest release tag, if available.